### ISSUE TITLE
Pilove for raspberry pi 4

### ISSUE THREAD

**Issue by LeonLenclos** [skarbat/pilove#5]
Hello, I know it's a five years old project but I'm taking a chance.

I tried to run pilove on a Raspberry 4. And it doesn't work. Although I don't understand why I guess it's normal.

Unfortunately I don't have the skills to make the necessary modifications myself (I have no idea what to do...).

Do you plan to port pilove to rpi4. If not, do you have any advice for me to do it myself?

Thanks you very much !

**Comment by JoKontage** [comment]
Sorry to necropost - you still trying to get it to work on an RPi4? If so, I can take a look at it.

**Comment by LeonLenclos** [comment]
I closed the issue because I found an alternative solution to launch a .love game at rpi startup using retropie.

Pilove would be better, cleaner, because of less useless packages around the game. But I'm not bothered by the performance or the boot speed with my current setup.

Nevertheless, if it's easy for you to take a look at the issue, I think a lot of people might be interested (including me for an update of this project, or other projects...).
